zerver/fixtures/crashlytics/crashlytics_issue_message.json

```diff
@@ -0,0 +1,13 @@
+{
+  "event": "issue_impact_change",
+  "payload_type": "issue",
+  "payload": {
+    "display_id": 123 ,
+    "title": "Issue Title" ,
+    "method": "methodName of issue",
+    "impact_level": 2,
+    "crashes_count": 54,
+    "impacted_devices_count": 16,
+    "url": "http://crashlytics.com/full/url/to/issue"
+  }
+}
```

Comment:
To be honest we can only have 2 types of messages - "issue message" - it's displayed and format the same way and verification message  (I'll add utest)  just for checking webhook.